src/src/components/App/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { commonText } from '../../localization/common';
+import { useBooleanState } from '../../hooks/useBooleanState';
+import { Portal } from '../Molecules/Portal';
+
+export function App() {
+  const [isOpen, _, handleClose, handleToggle] = useBooleanState();
```

Comment:
What is useBooleanState? Why not use useState react hook?

Reply:
I explained my reasoning in the documentation for that hook - https://github.com/maxxxxxdlp/calendar-stats/blob/1c4555747c45cb32c60907742aaa00faed188349/src/src/hooks/useBooleanState.tsx#L4-L36


**TLDR:** I find `onClick={handleClose}` easier to read than `onClick={(): void => setIsOpen(false)}`

But, if you think I should keep it simple I can revert it to `React.useState<boolean>();`

Reply:
All good appreciate the explanation!